### ISSUE TITLE
"no links loaded from stremio" cloud links error

### ISSUE THREAD

**Issue by oathzed** [Flixclusive/flx-providers#16]
Hello,

Im using the torentio addon, and I have links in my cloud and certain ones aren't playing from the catalog. Those exact same links play fine in stremio, it seems it can't get the link or something, this is the error:
![Screenshot_20240625-140540](https://github.com/Flixclusive/flx-providers/assets/52545913/b110aaf5-de49-423b-a802-977f4970fe70)

Some also give this error:

![Screenshot_20240625-141036](https://github.com/Flixclusive/flx-providers/assets/52545913/a98bc38f-4fbc-49e9-abc8-b2cfef870b7c)

Thanks!

**Comment by rhenwinch** [comment]
> I have links in my cloud and certain ones aren't playing from the catalog.

I lazily forgot about the movies. I only optimized the ones for TV Shows and Downloads folder caches

> they shouldn't have to have a TMDb I'd to be playable

This ones an app issue. Will fix on the next commits

**Comment by oathzed** [comment]
> > I have links in my cloud and certain ones aren't playing from the catalog.
> 
> I lazily forgot about the movies. I only optimized the ones for TV Shows and Downloads folder caches
> 
> > they shouldn't have to have a TMDb I'd to be playable
> 
> This ones an app issue. Will fix on the next commits

Ahh got yeh xD. Thanks a lot man 🙏

**Comment by oathzed** [comment]
> Hey,
> 
> Could you check if this issue still persists after v1.1.0?

Yes it seems to be working. Great job. The only thing that doesn't work but I guess not a huge deal is the "watch" button for tv show files, but if you click the actual episode it works. Might be nice to get that to work, if it's like a season pack or something you can just click watch and it will pick up where you left off. Currently it gives this error: 
![Screenshot_20240626-083739](https://github.com/Flixclusive/flx-providers/assets/52545913/53643ee2-f1b4-4cbe-9135-72e7815043b0)

**Comment by adilurrhmn** [comment]
Sane problem 
How to fix this?

![Uploading Screenshot_2024-07-03-01-32-24-84_e8ccd80afe35f7798e3674be308ad800.jpg…]()

![Screenshot_2024-07-03-01-35-53-18_e8ccd80afe35f7798e3674be308ad800](https://github.com/Flixclusive/flx-providers/assets/124466769/2037e62d-1986-45dd-8fef-712631b74287)
